ext/random/random.c

```diff
@@ -675,19 +675,20 @@ PHP_FUNCTION(lcg_value)
 PHP_FUNCTION(mt_srand)
 {
 	zend_long seed = 0;
+	bool seed_is_null = 1;
```

Comment:
```suggestion
	bool seed_is_null = true;
```
For consistency with `Mt19937::__construct()`.